src/ontology/edits/oeo-social.omn

```diff
@@ -243,24 +247,28 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1127",
 ObjectProperty: OEO_00020071
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A relation that holds between financial instruments, commodities, or other products, services, or goods and a market exchange.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A relation that holds between commodities and a market exchange."@en,
```

Comment:
In line with the suggestion for `is traded at` we should also add _... where the commodity is offered for trade/sale at the market exchange._ Else it would just say there is a relation between those two concepts, but does not specify the kind of the relations.